src/mockServer/mockData/items.ts

```diff
@@ -16,8 +16,8 @@ const MOCK_ITEMS: DiscriminatedItem[] = [
       },
     },
     creator: MOCK_MEMBERS[0],
-    createdAt: new Date(),
-    updatedAt: new Date(),
+    createdAt: Date.now().toString(),
```

Comment:
I think we should do:
```suggestion
    createdAt: new Date(Date.now()).toISOString(),
```
Because what you have here will just be the timestamp as a string instead of the ISO string of the date.